pilot/pkg/security/authn/utils/utils_test.go

```diff
@@ -39,6 +40,7 @@ func TestBuildInboundFilterChain(t *testing.T) {
 		node             *model.Proxy
 		listenerProtocol networking.ListenerProtocol
 		trustDomains     []string
+		tlsV2Disabled    bool
```

Comment:
not sure why you use `disabled`, positive predicate usually is easier to read and follow through.

Reply:
Completely agree. I was kind of trying to avoid changing all test cases. Fixed now.